src/commands/tileindex-validate/tileindex.validate.ts

```diff
@@ -223,10 +223,10 @@ export const commandTileIndexValidate = command({
     if (args.validate) {
       let validationFailed = false;
       for (const tiff of locations) {
-        const ret = validateTiffAlignment(tiff);
-        if (ret === true) continue;
-        logger.error({ reason: ret.message, source: tiff.source }, 'TileInvalid:Validation:Failed');
-        validationFailed = true;
+        const valid = validateTiffAlignment(tiff);
+        if (valid === false) {
+          validationFailed = true;
+        }
```

Comment:
I'd recommend using a single variable since we don't need `valid` except to update `validationFailed`. I'd also recommend using positive words in general, to avoid any double negatives. In other words, `validationFailed` could be something like `allValid`, and you could do something like `allValid = allValid && validateTiffAlignment(tiff);`